src/common/framework/logging/log_message.ts

```diff
@@ -1,21 +1,27 @@
 import { extractImportantStackTrace } from '../util/stack.js';
 
 export class LogMessageWithStack extends Error {
-  constructor(name: string, ex: Error, includeStack: boolean = true) {
+  printStack: boolean = true;
+  timesSeen: number = 1;
```

Comment:
Either comment that these members can and should be set externally, or maybe add some setters for them. Seems like we only ever increment `timesSeen` so we could have a method to do only that.